sdk/digitaltwins/Azure.DigitalTwins.Core/tests/E2eTestBase.cs

```diff
@@ -92,5 +93,17 @@ protected string GetRandom()
         {
             return Recording.GenerateId();
         }
+
+        protected async Task createAndListModelsAsync(DigitalTwinsClient client, List<string> lists)
```

Comment:
worth a comment we do this to work around a bug or issue of latency

Reply:
good idea